client/src/app/components/nova/StatusPill.scss

```diff
@@ -25,17 +25,17 @@
 
     &.status__ {
         &success {
-            background-color: var(--message-confirmed-bg);
-            color: $mint-green-7;
+            background-color: $block-confirmed;
```

Comment:
We should rethink this, so, we either create more classes specialized in blocks and transactions that put the pill that corresponds to the block state, same color as the visualizer for each state, and the rest of the pills in the explorer remain as they were before, of we undo the changes in this file for good